engine/runtime/src/main/java/org/enso/interpreter/runtime/callable/UnresolvedConstructor.java

```diff
@@ -63,16 +68,31 @@ String toDisplayString(boolean allowSideEffects) {
    * @return a object representing unresolved (constructor)
    */
   public static UnresolvedConstructor build(String name) {
-    return new UnresolvedConstructor(name, NONE);
+    return new UnresolvedConstructor(name, NONE, NONE);
   }
 
   /**
    * Marks this object as executable through the interop library.
    *
+   * @param addionalDescriptions description of the applied arguments
```

Comment:
nitpick, but if possible let's fix the typo:
```suggestion
   * @param additionalDescriptions description of the applied arguments
```

Reply:
Done in eb7b701aed